app/mailers/visitor_mailer.rb

```diff
@@ -21,27 +21,49 @@ def booking_confirmation_email(visit, confirmation, token)
     @confirmation = confirmation
     @token = token
 
-    mail(from: sender, reply_to: prison_mailbox_email, to: recipient, subject: "Visit confirmed: your visit for #{Date.parse(@slot.date).strftime('%e %B %Y').gsub(/^ /,'')} has been confirmed")
+    mail(
+      from: sender,
+      reply_to: prison_mailbox_email,
+      to: recipient,
+      subject: "Visit confirmed: your visit for " \
+       "#{Date.parse(@slot.date).strftime('%e %B %Y').gsub(/^ /,'')} "\
+       "has been confirmed"
```

Comment:
Further to this I <i>think</i> you can just remove having to put the subject and the localisation path in the mail. ActionMailer will do the lookup for you.